content/index.html

```diff
@@ -752,7 +872,32 @@ <h2>Evidence Quality – Superior, Strong, and Fair</h2>
             </i>
```

Comment:
Delete all of the follow from the list above:
•	The issuing source visually identified the applicant and performed further checks to confirm the existence of that person. 
•	The issuing process for the evidence ensured that it was delivered into the possession of the person to whom it relates. 
•	The evidence contains at least one reference number that uniquely identifies the person to whom it relates. 
•	The full name on the evidence must be the name that the person was officially known by at the time of issuance. Not permitted are pseudonyms, aliases, an initial for surname, or initials for all given names. 
•	The evidence contains a photograph of the person to whom it relates.